packages/ipfs-http-server/src/api/resources/object.js

```diff
@@ -505,7 +506,7 @@ exports.patchAppendData = {
       },
       query: {
         cid,
-        cidBase,
+        cidBase: base,
```

Comment:
Why was this necessary?  `cidBase` is more descriptive.

Reply:
@achingbrain problem was that `cidToString` on lines: 534, 540, 610, 625, 705, 711, 787, 793 were incorrectly passed `cidBase` option while function expects option named `base`. I could change all those calls to do `{ cid, base: cidBase ... }` but this seemed like smaller change.

Let me know if you want to keep this or change all the calls instead.